clj/src/cljd/build.clj

```diff
@@ -337,11 +339,14 @@
 
 (defn -main [& args]
   (let [f (java.io.File. (System/getProperty "user.dir") "cljd.edn")
+        is-windows? (= "windows" (str/lower-case (or (System/getProperty "os.name") "")))
         config (if (.exists f)
                  (with-open [rdr (-> f io/reader java.io.PushbackReader.)]
                    (edn/read rdr))
                  {})]
     (binding [*ansi* (and (System/console) (get (System/getenv) "TERM"))
+              *dart* (if is-windows? "dart.bat" *dart*)
```

Comment:
Note: I installed Dart using the official instructions on https://dart.dev/get-dart, and instead of "dart.bat" I have "dart.exe"

Reply:
The separately installed dart sdk does contain an entry for dart.exe, but the dart sdk provided by flutter executes the FLUTTER_PATH\bin\cache\dart-sdk\bin\dart.exe program through the dart.bat script.

In the command line, the dart command always works, but in the code, I need to delete the wrong dart binary under flutter/bin and add the dart sdk to the PATH before I can use dart.exe to find dart, which Not user friendly either.

Maybe look in the code for dart.exe (from Dart users) and dart.bat (from Flutter users) and find one of them.

Reply:
I would expect cljd work without flutter installed? I don't have flutter, hence I don't have dart.bat, only dart.exe...